lib/web_ui/lib/src/ui/canvas.dart

```diff
@@ -61,9 +61,9 @@ enum VertexMode {
 class Vertices {
   final VertexMode _mode;
   final Float32List _positions;
-  final Float32List _textureCoordinates;
+  final Float32List _textureCoordinates; // ignore: unused_field
```

Comment:
These don't look like they are fixing the warnings, but ignoring them. Is this intentional? Can we leave a comment explaining why it's OK to ignore them?

Reply:
Implemented in subclasses and removed unused_field.